Recrutify/Recrutify.DataAccess/Repositories/CandidateRepository.cs

```diff
@@ -37,6 +37,30 @@ public IQueryable<Candidate> GetUnassignedCandidatesByProject(Guid projectId)
                                                                                  || p.Status == Status.TechInterviewSecondStep)));
         }
 
+        public Task<List<Location>> GetlocationAsync(Guid? projectId = null)
+        {
+            if (projectId != null)
+            {
+                var filter = _filterBuilder.Where(c => c.ProjectResults.Any(c => c.ProjectId == projectId));
```

Comment:
Init it only once, using ? : operator, so you don't need if, and two filter vars and duplications

Reply:
done